langkit/compiled_types.py

```diff
@@ -754,6 +758,26 @@ def memoization_kind(self):
         """
         return (names.Name('Mmz') + self.name).camel_with_underscores
 
+    @property
+    def introspection_radix(self):
```

Comment:
introspection_root or introspection_root_name would sound more familiar to me :)